### ISSUE TITLE
Add percentage completeness to command line tool progress output

### ISSUE THREAD

**Issue by jlee-made** [schemaspy/schemaspy#119]
Hi

I appreciate schemaspy, it's really helpful to be able to quickly get an idea of where to start in a database you're not familiar with.

Because some databases with large numbers of tables, it would be useful to have a percentage completion indication rather than just progress indication dots.  I just ran an old schemaspy version  on a remote test database and it had a runtime of more than 4 minutes.  Anything enough to answer the question "will this ever finish" would be very reassuring :-)

I'm not sure how to tell what the version number of my old .jar is, there's no version in the manifest or exposed in the command-line UI that I can see.

**Comment by npetzall** [comment]
In those cases we now the number of things too do, this should be possible or something like processing table 9 of 899.

**Comment by danwatt** [comment]
Idea: https://github.com/ctongfei/progressbar

**Comment by npetzall** [comment]
I've been playing with Jansi and logback. I think I have something usable.

**Comment by npetzall** [comment]
Yeah, Linux ok, older windows, newer windows not so much, gonna look into jn(i/a) for windows.

**Comment by npetzall** [comment]
Actually, was a ansi command that didn't work on windows. Switched to another and it worked. Have started the implementation.

Progress Lines will be shown at the bottom and log rows will be added above. Will support multiple progress lines.

**Comment by rafalkasa** [comment]
@npetzall awesome news I am interested about the results of your work...

**Comment by npetzall** [comment]
https://github.com/npetzall/schemaspy/tree/119_progressbar

It only contains progressbar/line for XMLOutput and Database reading (DatabaseService).

It works on linux and windows 

Here's a screentshot while running:
![image](https://user-images.githubusercontent.com/4175123/36515227-93cbd038-1778-11e8-86e6-c4b97d84e6aa.png)

And when finished (different run since connecting tables takes for ever as you can see.
![image](https://user-images.githubusercontent.com/4175123/36515317-f5c1fac4-1778-11e8-812b-9976e81f7c70.png)

I welcome any feedback or suggestions.
